documentation/decisions/0006-unit-catalog.md

```diff
@@ -0,0 +1,44 @@
+# Unit Catalog
```

Comment:
Unit Catalog: Use UNECE Recommendation 20 with ~ 1800 units.

Can we write "UN/ECE"? Then people understand more easily it's from the UN.

Reply:
We could, but we shouldn't, since [UNECE](https://en.wikipedia.org/wiki/United_Nations_Economic_Commission_for_Europe) is the official spelling.